importing/generate-legacy-resource-id-inserts.js

```diff
@@ -0,0 +1,232 @@
+// via a query on lara2 rds instance on 7/21/2002 
+
+const oldLaraFirestoreInfo = require("./prod-firestore.json")
+const nameToIdMap = {}
+
+oldLaraFirestoreInfo.forEach(item => {
+	// skip duplicate unused glossary
```

Comment:
The other glossary with the same name isn't used in any activities.  The `prod-joined-glossaries.txt` saved data from when I migrated showed that is isn't used.